src/tabs/tab.jsx

```diff
@@ -5,19 +5,58 @@ import ThemeManager from '../styles/theme-manager';
 
 const Tab = React.createClass({
 
-  mixins: [StylePropable],
+  mixins: [
+    StylePropable,
+  ],
 
   contextTypes: {
     muiTheme: React.PropTypes.object,
   },
 
   propTypes: {
+    /**
+     * The css class name of the root element.
+     */
+    className: React.PropTypes.string,
+
+    /**
+     * Sets the text value of the tab item to the string specified.
+     */
     label: React.PropTypes.node,
+
+    /**
+     * Fired when the active tab changes by touch or tap.
+     * Use this event to specify any functionality when an active tab changes.
+     * For example - we are using this to route to home when the third tab becomes active.
+     * This function will always recieve the active tab as it\'s first argument.
+     */
     onActive: React.PropTypes.func,
+
+    /**
+     *
+     */
     onTouchTap: React.PropTypes.func,
+
+    /**
+     * Defines if the current tab is selected or not.
+     * The Tabs component is responsible for setting this property.
+     */
     selected: React.PropTypes.bool,
+
+    /**
+     * Override the inline-styles of the root element.
+     */
     style: React.PropTypes.object,
+
+    /**
+     * If value prop passed to Tabs component, this value prop is also required.
+     * It assigns a value to the tab so that it can be selected by the Tabs.
+     */
     value: React.PropTypes.string,
```

Comment:
i believe value must be `any` so you don't have to parseInt for the swipeable view,